.storybook/preview-head.html

```diff
@@ -7,13 +7,13 @@
               var srcUrl = null;
               var host = location.hostname || 'localhost';
               var defaultPort = location.protocol === 'https:' ? 443 : 80;
-              var port = 4200;
+              var port = 7020;
               var path = '';
               var prefixURL = '';
               var src = srcUrl || prefixURL + '/_lr/livereload.js?port=' + port + '&host=' + host + path;
               var script = document.createElement('script');
               script.type = 'text/javascript';
-              script.src = location.protocol + '//' + host + ':4200' + src;
+              script.src = location.protocol + '//' + host + ':7020' + src;
```

Comment:
meh, this is to conflict 100% of the time on my machine ; how about 4201?

Reply:
I remove the file and set in the `.gitignore`. It will automatically generate